dirty.rb

```diff
@@ -0,0 +1,38 @@
+a =     'b'
+eval '1+1'
+"#{1}"
+begin
+  foo
+rescue
+end
+begin
+begin
+begin
```

Comment:
__[staging][rubocop]__
* Use 2 (not 0) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)